pallets/collator-selection/src/lib.rs

```diff
@@ -136,19 +141,26 @@ pub mod pallet {
 		/// Used only for benchmarking.
 		type MaxInvulnerables: Get<u32>;
 
-		// Will be kicked if block is not produced in threshold.
-		type KickThreshold: Get<Self::BlockNumber>;
+		// n-th Percentile of lowest-performing collators to be checked for kicking
+		type PerformancePercentileToConsiderForKick: Get<u8>;
```

Comment:
oh yeah, makes sense to do it in the type definition instead of during the computation.

Reply:
> Is it a fixed value, or can we add a call to modify it?

At the moment it is a runtime config variable that is hard-set at genesis.
I agree that it would make sense to allow for later changes by adding a callable to mutate it.
Working on it

Reply:
This looks resolved now ?

Reply:
Is resolved, new setters are here: https://github.com/Manta-Network/Manta/pull/358/files#diff-a0359a0add814d66aac74493f7e17aefdcee21d3de1ae592e97059082b6feedfR374